cypress/e2e/spec.cy.js

```diff
@@ -87,3 +87,46 @@ describe("window properties", () => {
     cy.title().should("not.contain", "Reading App");
   });
 });
+
+// ✅ --- PASSING ---
+describe("Local storage", () => {
+  it("Check if input name is stored", () => {
+    cy.visit("/learner-onboarding/name");
+    cy.get("#learnerName")
+      .should("be.visible")
+      .type("Potato")
+      .should(() => {
+        expect(localStorage.getItem("learner-name")).to.eq('"Potato"');
+      });
+    cy.visit("/child-landing");
+    cy.get("h2").should("not.contain", "Hi, there");
+    cy.contains("Hi, Potato!");
+  });
+});
+
+// ✅ --- PASSING ---
+describe("Does the data matches", () => {
```

Comment:
great to see that your testing if the data is there to match each 8 colours, well done 🧪 ✨